packages/sanity/src/components/previews/general/DefaultPreview.tsx

```diff
@@ -8,7 +8,8 @@ import {PREVIEW_MEDIA_SIZE} from '../constants'
 import {PreviewMediaDimensions, PreviewProps} from '../types'
 import {renderPreviewNode} from '../helpers'
 
-export interface DefaultPreviewProps extends Omit<PreviewProps<'default'>, 'schemaType'> {
+export interface DefaultPreviewProps
+  extends Omit<PreviewProps<'default'>, 'schemaType' | 'renderDefault'> {
```

Comment:
Do we need to omit the `schemaType` here?

Reply:
True, it is not necessary – will remove that omit 👍